### DISCUSSION TITLE
How to connect discord bot with webhost

### DISCUSSION THREAD

**Discussion by Mr4sR** [Rapptz/discord.py#6320] [Q&A]
Can we use Discord bot we can write to .php file in web hosting?

**Comment by oliver-ni** [comment]
I can't tell what you mean by "write to .php file in web hosting". You need to be more specific. Do you mean something like dynamically update the contents of a web page based on some state in your Discord bot? If that's the case, you should ask how to do *that*, not what you just did.

https://xyproblem.info/
